lib/gui/pages/settings/controllers/settings.js

```diff
@@ -55,6 +55,13 @@ module.exports = function(WarningModalService, ErrorService, AnalyticsService) {
   this.model = settings;
 
   /**
+   * @summary Environment
+   * @type {Object}
+   * @public
+   */
+  this.environment = process.env;
```

Comment:
I wonder if it would be better to _just_ expose `ETCHER_DISABLE_UPDATES`, rather than exposing the whole `process.env` ?